chassis_action_name: "chassis_actions"
front_loader_action_name: "front_loader_actions"
gripper_action_name: "gripper_actions"
move_base_namespace: "/move_base"

global_frame: "map"
robot_frame: "base_link"
gripper_frame: "gripper_link"
tilt_base_frame: "tilt_base_link"
linear_frame: "linear_link"

gripper_max_dist: 0.112
max_vel_x: 0.15
max_vel_theta: 1.0

goal_distance_offset: 0.1
near_object_distance: 0.5
local_costmap_width: 3.0
replan_distance: 0.25

pickup_z_offset: 0.02
deliver_z_offset: 0.02

transport_z_height: 0.08

fast_stepper_speed: 0.044
slow_stepper_speed: 0.02

plow_into_object_offset: 0.0
